packages/lib/src/logger.ts

```diff
@@ -86,6 +88,34 @@ export class LogManager {
 	}
 }
 
+const defaultLoggingConfig: LoggingConfig = {
+	appenders: {
+		smartthings: { type: 'file', filename: 'smartthings.log' },
+		stderr: { type: 'stderr' },
+		errors: { type: 'logLevelFilter', appender: 'stderr', level: 'error' },
+	},
+	categories: {
+		default: { appenders: ['smartthings', 'errors'], level: 'warn' },
+		'rest-client': { appenders: ['smartthings', 'errors'], level: 'warn' },
+		cli: { appenders: ['smartthings', 'errors'], level: 'warn' },
+	},
+}
+
+
+const loggingDocsLink = 'https://github.com/SmartThingsCommunity/' +
+	'smartthings-cli/blob/master/packages/cli/doc/configuration.md#logging'
+export function loadLoggingConfig(filename: string): LoggingConfig {
+	if (!fs.existsSync(filename)) {
+		return defaultLoggingConfig
+	}
+
+	const parsed = yaml.safeLoad(fs.readFileSync(`${filename}`, 'utf-8'))
```

Comment:
Can't the first argument just be `filename`? Does it have to be a template literal here for some reason?

Reply:
It shouldn't be...probably copied code from where one was and didn't notice I removed everything except the `filename`. Fixed.